tests/konflux-demo/config/scenarios.go

```diff
@@ -15,7 +15,7 @@ var ApplicationSpecs = []ApplicationSpec{
 		ComponentSpec: ComponentSpec{
 			Name:                       "konflux-demo-component",
 			Language:                   "Java",
-			GitSourceUrl:               fmt.Sprintf("https://github.com/%s/%s", utils.GetEnv(constants.GITHUB_E2E_ORGANIZATION_ENV, "redhat-appstudio-qe"), "hacbs-test-project"),
+			GitSourceUrl:               fmt.Sprintf("https://github.com/%s/%s", utils.GetEnv(constants.GITHUB_E2E_ORGANIZATION_ENV, "redhat-appstudio-qe"), "hacbs-test-project-konflux-demo"),
```

Comment:
was this change intentional or it's also a temporary one?

Reply:
it was intentional one.. since `jvm-build-service`  tests uses the [same repo hacbs-test-project](https://github.com/konflux-ci/e2e-tests/blob/995cc017c91794609896078f18141e9596e48678/tests/build/jvm-build.go#L33) , so I forked and renamed for avoiding conflict during component creation